cprover_bindings/src/goto_program/location.rs

```diff
@@ -81,6 +121,27 @@ impl Location {
         Location::Loc { file, function, line, col }
     }
 
+    pub fn property_location<T, U: Into<InternedString>, V: Into<InternedString>>(
+        file: U,
+        function: Option<V>,
+        line: T,
+        col: Option<T>,
+        comment: Option<U>,
+        property_name: Option<U>,
```

Comment:
IMO, we should either keep this function simple or flexible. This implementation is neither. You should either add a 4th parameter here (like `X: Into<InternedString>`) or just keep them all as one type. I.e.:  `pub fn property_location<T, U: Into<InternedString>>`

Reply:
Will do!